tensorflow_datasets/core/visualization/as_dataframe.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""As dataframe util.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from typing import Any
+
+import tensorflow.compat.v2 as tf
+from tensorflow_datasets.core import dataset_info
+from tensorflow_datasets.core.visualization import dataframe_visualizer
+
+
+_ALL_VISUALIZERS = [
+    dataframe_visualizer.DataframeVisualizer(),
```

Comment:
In this case, as we only expect a single visualizer, it seems overkill to add this extra class. The code could go directly in `as_dataframe` (Not use `visualizer.Visualizer` at all and remove `dataframe_visualizer.py` entirely)

Reply:
Fixed.